engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -148,6 +148,8 @@ object ProcessCompilationError {
 
   case class MismatchParameter(message: String, description: String, paramName: String, nodeId: String) extends ParameterValidationError
 
+  case class InvalidNumberParameter(message: String, description: String, paramName: String, nodeId: String) extends ParameterValidationError
```

Comment:
Please use InvalidPropertyFixedValue. We don't want errors for every single case here.